### PR TITLE
Adds i18n for email

### PR DESCRIPTION
**Pull request by Drocheg**
-Adds internationalization for emails with english and spanish
-Adds locale column to users to have the language for that user

### CODE REVIEW

**Comment by Drocheg** [comment]
https://trello.com/c/Iz8KpBc6/27-internacionalizar-el-email

**Comment by Drocheg** [comment]
image

image

File: app/mailers/rent_mailer.rb [hunk 1]
```diff
@@ -8,6 +8,11 @@ class RentMailer < ApplicationMailer
   #
   def new_rent_notification(rent_id)
     @rent = Rent.find(rent_id)
-    mail to: @rent.user.email, subject: 'Success! You did it.'
+    I18n.with_locale(@rent.user.locale) do
+      mail(
+        to: @rent.user.email,
+        subject: default_i18n_subject
```
Comment (Gmarn) [inline app/mailers/rent_mailer.rb]: Seems to work but I do not like this so much ¬¬ not very clear...

**Review by Gmarn — APPROVED** [review]
A lot of unused translations was added you could remove them. If you want.

**Review by Gmarn — APPROVED** [review]

**Pull request merged**